Controller/MailgunEventController.php

```diff
@@ -116,6 +116,7 @@ public function indexAction(Request $request, $page, $pageSize)
 
         // get the events
         $params['events'] = $this->getRepository()->getEvents($filter, array($orderBy => $orderDirection), $pageSize, ($page - 1) * $pageSize);
+        $params['emailWebViewRoute'] = $this->getParameter(AzineMailgunWebhooksExtension::PREFIX.'_'.AzineMailgunWebhooksExtension::SENT_EMAIL_ADMIN_OVERVIEW_ROUTE);
```

Comment:
The feature should also be usable without the use of the AzineEmailBundle.
=> Rename the param 'SENT_EMAIL_ADMIN_OVERVIEW_ROUTE' and 'sent_email_admin_overview_route' to 'WEB_VIEW_ROUTE" and "web_view_route" in all files.

Also add a second config-option ('web_view_token') to change the key-name of the token. Default to "x-azine-webview-token", as this is the new mail-header we add.

Also update the README.md with information about this configuration options.